src/bootstrap/configure.py

```diff
@@ -521,13 +521,19 @@ def write_config_toml(writer, section_order, targets, sections):
         else:
             writer = write_uncommented(sections[section], writer)
 
+def quit_if_file_exists(file):
+    if os.path.isfile(file):
+        p("Existing '" + file + "' detected. EXITING")
+        quit()
```

Comment:
Please exit with a non-zero exit code.
```suggestion
        err("Existing '" + file + "' detected.")
```